### PR TITLE
Fixes a few runtimes

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Fixes some runtimes, most of which were caused by the verb PR

## Why It's Good For The Game
These shouldn't runtime and should actually work.

## Testing
Could do all of the things without runtiming.

## Changelog
:cl:
fix: Fixed not being able to take beakers out of sleepers
fix: You can now unlock crates with altclick again
/:cl:

### CODE REVIEW

File: code/game/machinery/Sleeper.dm [hunk 1]
```diff
@@ -295,7 +295,7 @@
 			else
 				to_chat(usr, "<span class='danger'>This person is not in good enough condition for sleepers to be effective! Use another means of treatment, such as cryogenics!</span>")
 		if("removebeaker")
-			remove_beaker()
+			remove_beaker(usr)
```
Comment (Contrabang) [inline code/game/machinery/Sleeper.dm:298]:
```suggestion
			remove_beaker(ui.user)
```
you'll need to fix the ui_act call to be
```
ui_act(action, params, datum/tgui/ui)
```

File: code/modules/paperwork/paper.dm [hunk 2]
```diff
@@ -102,7 +102,6 @@
 		return ..()
 
 /obj/item/paper/proc/rename(mob/user)
-
 	if(HAS_TRAIT(user, TRAIT_CLUMSY) && prob(50))
```
Comment (Contrabang) [inline code/modules/paperwork/paper.dm:105]:
```suggestion
	if(user && HAS_TRAIT(user, TRAIT_CLUMSY) && prob(50))
```

File: code/modules/surgery/organs_internal.dm [hunk 3]
```diff
@@ -820,8 +820,7 @@
 		"[user] is beginning to cauterize the incision on [target]'s [zone] with \the [tool].",
 		"You are beginning to cauterize the incision on [target]'s [zone] with \the [tool]."
 	)
-	var/obj/item/organ/affected = target.get_organ(target_zone)
-	affected.custom_pain("Your [zone] is being burned!")
+	to_chat(user, "<span class='userdanger'>Your carapace is being burned!</span>") // No custom pain because xenos are special
```
Comment (Contrabang) [inline code/modules/surgery/organs_internal.dm:823]: Why not use the zone_name proc? isnt that what its for?

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/paper.dm [hunk 4]
```diff
@@ -102,8 +102,7 @@
 		return ..()
 
 /obj/item/paper/proc/rename(mob/user)
-
-	if(HAS_TRAIT(user, TRAIT_CLUMSY) && prob(50))
+	if(user && HAS_TRAIT(user, TRAIT_CLUMSY) && prob(50))
```
Comment (Contrabang) [inline code/modules/paperwork/paper.dm:105]: Actually, my bad, this user check should be an early return, since the rest of the proc uses user too

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/paper.dm (hunk 4 above)
Comment (DGamerL) [reply code/modules/paperwork/paper.dm:105]: Removed the check, it was broken even more deeply and GDN would fix it

**Review by Contrabang — APPROVED** [review]
LGTM

**Review by Sirryan2002 — APPROVED** [review]

**Pull request merged**